activeadmin.gemspec

```diff
@@ -21,11 +21,11 @@ Gem::Specification.new do |s|
   s.add_dependency 'arbre', '>= 1.1.1'
   s.add_dependency 'formtastic', '~> 3.1'
   s.add_dependency 'formtastic_i18n'
-  s.add_dependency 'inherited_resources', '~> 1.7'
+  s.add_dependency 'inherited_resources', '>= 1.7.0'
   s.add_dependency 'jquery-rails', '>= 4.2.0'
   s.add_dependency 'kaminari', '>= 1.0.1'
   s.add_dependency 'railties', '>= 4.2', '< 5.2'
```

Comment:
This was supposed to be for Rails 5.2 support but you're specifying railties < 5.2. How's that supposed to work? Rails 5.2 requires railties 5.2.0. https://github.com/rails/rails/commit/375a4143cf5caeb6159b338be824903edfd62836#diff-e79a60dc6b85309ae70a6ea8261eaf95R81